src/current/cockroachcloud/cloud-api.md

```diff
@@ -501,54 +596,71 @@ If the request was successful, the client will receive a list of all clusters wi
 {
   "clusters": [
     {
+      "account_id": "",
       "cloud_provider": "{cloud_provider}",
-      "created_at": "2022-03-14T14:15:22Z",
+      "cockroach_version": "v23.2.4",
+      "config": {
+        "serverless": {
+          "routing_id": "{routing_id}",
+          "upgrade_type": "{upgrade_type}",
+          "usage_limits": {
+            "provisioned_virtual_cpus": "2"
+          }
+        }
+      },
+      "created_at": "2022-03-22T20:23:11.285067Z",
       "creator_id": "{account_id}",
-      "deleted_at": "2022-03-14T14:15:22Z",
       "id": "{cluster_id}",
-      "operation_status": "CLUSTER_STATUS_UNSPECIFIED",
       "name": "{cluster_name}",
-      "plan": "SERVERLESS",
+      "operation_status": "UNSPECIFIED",
+      "plan": "STANDARD",
       "regions": [
         {
+          "internal_dns": "{internal_dns}",
           "name": "{region_name}",
+          "node_count": 0,
+          "private_endpoint_dns": "{private_endpoint_dns}",
           "sql_dns": "{server_host}",
           "ui_dns": ""
         }
       ],
-      "config": {
-        "serverless": {
-          "regions": [
-            "{region_name}"
-          ],
-          "spend_limit": {spend_limit},
-          "routing_id": "{routing_id}"
-        }
-      },
-      "state": "CREATING",
-      "updated_at": "2022-03-14T14:15:22Z"
+      "state": "CREATED",
+      "updated_at": "2022-03-22T20:23:11.879593Z",
+      "upgrade_status": "UPGRADE_AVAILABLE"
     }
   ],
-  ...
+  "pagination": {
+    "next_page": "<string>",
+    "previous_page": "<string>"
   }
 }
 ~~~
 
 Where:
 
+  - `{cloud_provider}` is the name of the cloud infrastructure provider. Possible values are: `GCP`, `AWS`, `AZURE`.
```

Comment:
If we must show the example output, we should not document every field in it as replaceable. I also notice here that we use angle brackets for some replaceable content, but the examples here all use curly braces.

Reply:
Dropping these, but adding a follow up task to standardize field definition format